app/main.py

```diff
@@ -1,25 +1,92 @@
 from abc import ABC
+from typing import Any
 
 
 class IntegerRange:
-    pass
+
+    def __init__(
+            self,
+            min_amount: int,
+            max_amount: int
+    ) -> None:
+
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: Any, name: str) -> None:
+        self.private_name = "_" + name
+
+    def __get__(self, instance: object, owner: Any) -> float:
+        return getattr(instance, self.private_name)
+
+    def __set__(self, instance: object, value: float) -> float | None:
+        if self.min_amount <= value <= self.max_amount:
+            return setattr(instance, self.private_name, value)
+        raise ValueError(f"Value must be"
+              f"{self.min_amount}...{self.max_amount}")
 
 
 class Visitor:
-   pass
+
+    def __init__(
+            self,
+            name: str,
+            age: int,
+            weight: float,
+            height: float) -> None:
```

Comment:
Move `) -> None:` to a new line so that opening and closing parentheses will not have any arguments on their lines.